legacy/firmware/ethereum.c

```diff
@@ -565,7 +570,7 @@ static void ethereum_signing_handle_erc20(struct signing_params *params) {
       memcmp(params->data_initial_chunk_bytes,
              "\xa9\x05\x9c\xbb\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00",
              16) == 0) {
-    params->token = tokenByChainAddress(chain_id, params->pubkeyhash);
+    params->token = defs ? &defs->token : UnknownToken;
```

Comment:
...and unconditionally stick it into `params->token`, the `UnknownToken` should be there from much sooner.

Reply:
9556b8f18d0dab6fc84e20f31bad8ba707737562